docs/howto/run_material_mapping.md

```diff
@@ -0,0 +1,119 @@
+
+
+This configuration can be cumbersome to do by hand especially when trying to map on sensitives surfaces. To simplify this task two python script are available in Examples/scripts/MaterialMaping :
+
+*  writeMapConfig.py
+*  configureMap.py
+
+The first one take as an input the surfaces map previously generated and will return a json config file. In this file you can see all the different type of surfaces associated to each volume. You can then change the binning associated to a surface type, when the second script is called the resulting map will have the binning applied to all the surface of the corresponding type and `"mapMaterial"` will be change to true. Attention : the configureMap.py will modify the surfaces map used in input you might want to make a copy of it beforehand.
+
+
+```
+python3 ../acts-framework/scripts/MaterialMaping/writeMapConfig.py geometry-map.json config-map.json
+```
+
+Then edit the config-map.json file
+
+```
+../acts-framework/scripts/MaterialMaping/configureMap.py geometry-map.json config-map.json
+```
+
+## Geantino scan
+
+The next step is to do a geantino scan of our detector. For this we will use the GeantinoRecording algorithm :
+
+```
+./../build/bin/ActsSimGeantinoRecording -j1 --dd4hep-input ../Examples/Detectors/DD4hepDetector/compact/OpenDataDetector/OpenDataDetector.xml --output-root true -n10000
+```
+
+The result of the geantino scan will be a root file containing Material Tracks, those contain the direction and production vertex of the geantino, the total material accumulated and all the interaction point in the detector.
+
+## Material Mapping
+
+With the surfaces map and the material track we can finally do the material mapping using the MaterialMapping algorithm :
+
+```
+./../build/bin/ActsExampleMaterialMappingDD4hep -j1 --input-root true --input-files geant-material-tracks.root --mat-input-type file --mat-input-file geometry-map.json --output-root true --output-json true --mat-mapping-collection material-tracks --mat-output-file material-maps --mat-mapping-surfaces true --mat-mapping-volumes true --mat-mapping-volume-stepsize 1 --dd4hep-input ../Examples/Detectors/DD4hepDetector/compact/OpenDataDetector/OpenDataDetector.xml
+```
+
+As an output you will obtain the material map as a root and json file and a new Material Tracks collection in a root file. This new collection add to each material interaction the associated surface during the mapping. This can be used for the control plots.
+Depending on what you want to do there are three option you can change :
+*  `mat-mapping-surfaces` : determine if material is mapped onto surfaces
+*  `mat-mapping-volumes` : determine if material is mapped onto volumes
+*  `mat-mapping-volume-stepsize` : Determine the step size used in the sampling of the volume this should be small compared to you bin size.
```

Comment:
```suggestion
As an output you will obtain the material map as a root and JSON file and a new material track collection in a root file. This new collection adds to each material interaction the associated surface during the mapping. This can be used for the control plots.
Depending on what you want to do there are three option you can change :
*  `mat-mapping-surfaces` : determine if material is mapped onto surfaces
*  `mat-mapping-volumes` : determine if material is mapped onto volumes
*  `mat-mapping-volume-stepsize` : determine the step size used in the sampling of the volume. This should be small compared to the bin size.
```